utils/treesitter/queries/highlights.scm

```diff
@@ -0,0 +1,124 @@
+; Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+; Exceptions. See /LICENSE for license information.
+; SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
```

Comment:
I'm not sure how this file relates to the other files. Could you please add comments explaining?